server/schedulers/utils.go

```diff
@@ -358,3 +199,76 @@ func summaryPendingInfluence(pendings map[*pendingInfluence]struct{}, f func(*op
 	}
 	return ret
 }
+
+type storeLoad struct {
+	ByteRate float64
+	Count    int
+}
+
+func (load *storeLoad) ToLoadPred(infl Influence) *storeLoadPred {
+	future := *load
+	future.ByteRate += infl.ByteRate
+	return &storeLoadPred{
+		Current: *load,
+		Future:  future,
+	}
+}
+
+// store load prediction
+type storeLoadPred struct {
+	Current storeLoad
+	Future  storeLoad
+}
+
+func (lp *storeLoadPred) min() storeLoad {
+	return minLoad(&lp.Current, &lp.Future)
+}
+
+func (lp *storeLoadPred) max() storeLoad {
+	return maxLoad(&lp.Current, &lp.Future)
+}
+
+func minLoad(a, b *storeLoad) storeLoad {
```

Comment:
Would it return a `storeLoad` with `a.ByteRate` and `b.Count`, does this work as expected?

Reply:
Yes, expected.